indico/web/rh.py

```diff
@@ -331,6 +331,9 @@ def process(self):
             res = ''
 
         response = current_app.make_response(res)
+        if hasattr(self, 'noindex'):
```

Comment:
If you want to put this here, set `self.noindex = False` in `__init__` and check for `self.noindex` here.

Alternatively, override this method in `RHDisplayEventBase` and add the header after the super call.

Reply:
Done and changed hasattr in getattr